cmd/snap/cmd_debug_disks.go

```diff
@@ -0,0 +1,60 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```

Comment:
did you really mean to add this file? There are a few other changes that I suspect are not intentional :-)